docs/guides/Package-authoring-best-practices.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Package authoring best practices
+description: A general guide of best practices for creating high quality NuGet packages.
+author: chgill-MSFT
+ms.author: chgill
+ms.date: 09/17/2020
+ms.topic: conceptual
+---
+
+# Package authoring best practices
+
+This guidance is intended to give NuGet.org package authors a lightweight end to end reference for creating and publishing high quality packages. It will primarily focus on package-specific best practices such as metadata and packing. For more in-depth suggestions for building high quality libraries, see the .NET [Open-source library guidance](https://docs.microsoft.com/dotnet/standard/library-guidance/).
+
+## Types of recommendations
+
+Each article presents four types of recommendations: **Do**, **Consider**, **Avoid**, and **Do not**. The type of recommendation indicates how closely it should be followed.
+
+You should almost always follow a **Do** recommendation. For example:
+
+✔️ DO include a short description for your package that describes what it's for.
+
+On the other hand, **Consider** recommendations should generally be followed, but there are legitimate exceptions to the rule:
+
+✔️ CONSIDER choosing a NuGet package name with a prefix that meets NuGet's prefix reservation [criteria](https://docs.microsoft.com/nuget/reference/id-prefix-reservation).
+
+**Avoid** recommendations mention things that are generally not a good idea, but breaking the rule sometimes makes sense:
+
+❌ AVOID NuGet package references that demand an exact version.
+
+And finally, **Do not** recommendations indicate something you should almost never do:
+
+❌ DO NOT use the `LicenseUrl` metadata property.
+
+## Create a NuGet package
+
+The latest recommended way to to create a NuGet package is from an [SDK-style project](https://docs.microsoft.com/nuget/resources/check-project-format). SDK-style project properties, including target [framework](#framework-targeting) and [package metadata](#package-metadata), are defined in the project file (.csproj).
```

Comment:
`.csproj` is C# specific. Can we say `(.csproj or .fsproj)` instead? (I've never created an F# package before, please double-check this is correct)

Reply:
Can we simplify and just say ".NET SDK-style projects"?

Reply:
or link to https://docs.microsoft.com/en-us/visualstudio/ide/solutions-and-projects-in-visual-studio?view=vs-2019#project-file

Reply:
there's too many project files to name here. .csproj is the most used, but i think sdk-style project files is most correct.

Reply:
```suggestion
The latest recommended way to to create a NuGet package is from an [SDK-style project](https://docs.microsoft.com/nuget/resources/check-project-format). SDK-style project properties, including target [framework](#framework-targeting) and [package metadata](#package-metadata), are defined in the [project file](https://docs.microsoft.com/visualstudio/ide/solutions-and-projects-in-visual-studio?view=vs-2019#project-file).
```